src/index.html

```diff
@@ -22,6 +22,319 @@
+        <img
+          class="card__img"
+          src="./images/imac.jpeg"
+          alt="APPLE A1419 iMac 27 Retina"
+        />
+        <div class="card__description">
+          <h2 class="card__title">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h2>
+          <p class="card__code">Product code: 195434</p>
+        </div>
+        <div class="card__reviews">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <p class="card__reviews-count">Reviews: 5</p>
+        </div>
+        <div class="card__price">
+          <p class="card__price-text">Price:</p>
+          <p class="card__dollar">$2,199</p>
+        </div>
+        <button class="card__button">Buy</button>
+      </article>
+      <article class="card">
+        <img
+          class="card__img"
+          src="./images/imac.jpeg"
+          alt="APPLE A1419 iMac 27 Retina"
+        />
+        <div class="card__description">
+          <h2 class="card__title">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h2>
+          <p class="card__code">Product code: 195434</p>
+        </div>
+        <div class="card__reviews">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <p class="card__reviews-count">Reviews: 5</p>
+        </div>
+        <div class="card__price">
+          <p class="card__price-text">Price:</p>
+          <p class="card__dollar">$2,199</p>
+        </div>
+        <button class="card__button">Buy</button>
+      </article>
+      <article class="card">
+        <img
+          class="card__img"
+          src="./images/imac.jpeg"
+          alt="APPLE A1419 iMac 27 Retina"
+        />
+        <div class="card__description">
+          <h2 class="card__title">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h2>
+          <p class="card__code">Product code: 195434</p>
+        </div>
+        <div class="card__reviews">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <p class="card__reviews-count">Reviews: 5</p>
+        </div>
+        <div class="card__price">
+          <p class="card__price-text">Price:</p>
+          <p class="card__dollar">$2,199</p>
+        </div>
+        <button class="card__button">Buy</button>
+      </article>
```

Comment:
The product cards are repeated multiple times with the same content. This repetition can be optimized by using a templating engine or JavaScript to dynamically generate these cards. This will make the code more maintainable and easier to update.